.github/workflows/tag.yml

```
@@ -37,6 +37,10 @@ jobs:
       - name: Build
         run: npm run ${{ steps.variables.outputs.BUILD_SCRIPT }} -- --env release=true --env basename="v/${{ steps.variables.outputs.VERSION }}" --env semver=${{ steps.variables.outputs.VERSION }} --env info_version=${{ steps.variables.outputs.VERSION }} --env github_actions=true
 
+      - name: Create latest version reference file
+        run: |
+          echo "{ \"latestVersion\" : \"${{ steps.variables.outputs.BRANCH }}\" }" > ./dist/latestVersion.json
```

should it be `VERSION` instead of  `BRANCH` when on PROD ?
```suggestion
          echo "{ \"latestVersion\" : \"${{ steps.variables.outputs.VERSION }}\" }" > ./dist/latestVersion.json
```

---

Yeah, you are totally right. Fixing it.